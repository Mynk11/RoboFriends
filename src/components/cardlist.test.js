import React from 'react';

import { shallow, mount, render } from 'enzyme';
import 'tachyons';
import Cardlist from './cardlist';
const mock = [{
    id: 1,
    name: "John Snow",
    email: "john@snow.com"
},
{
    id: 2,
    name: "Jon now",
    email: "jon@now.com"
}
]


it("Should run ", () => {
    //expect(shallow(<Cards />).length).toEqual(1);
    expect(shallow(<Cardlist robots={mock} />)).toMatchSnapshot();
});
