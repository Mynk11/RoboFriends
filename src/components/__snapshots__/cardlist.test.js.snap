// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should run  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <cardlist
    robots={
      Array [
        Object {
          "email": "john@snow.com",
          "id": 1,
          "name": "John Snow",
        },
        Object {
          "email": "jon@now.com",
          "id": 2,
          "name": "Jon now",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <cards
          email="john@snow.com"
          id={1}
          name="John Snow"
        />,
        <cards
          email="jon@now.com"
          id={2}
          name="Jon now"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {
          "email": "john@snow.com",
          "id": 1,
          "name": "John Snow",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "function",
        "props": Object {
          "email": "jon@now.com",
          "id": 2,
          "name": "Jon now",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <cards
            email="john@snow.com"
            id={1}
            name="John Snow"
          />,
          <cards
            email="jon@now.com"
            id={2}
            name="Jon now"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {
            "email": "john@snow.com",
            "id": 1,
            "name": "John Snow",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "function",
          "props": Object {
            "email": "jon@now.com",
            "id": 2,
            "name": "Jon now",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
